src/app/components/MediaLoader/utils/buildSetting.test.ts

```diff
@@ -397,4 +400,95 @@ describe('buildSettings', () => {
       );
     });
   });
+
+  describe('OnDemandTv', () => {
+    const hindiTvBaseSettings = {
+      counterName: 'hindi.bbc_hindi_tv.tv.w172zm8920nck2z.page',
+      lang: 'hi',
+      service: 'hindi' as Services,
+      statsDestination: 'WS_NEWS_LANGUAGES',
+      producer: 'HINDI',
+      translations: hindiServiceConfig.default.translations,
+    } as BuildConfigProps;
+
+    const hindiTvMediaBlocks = hindiTvProgramme.content.blocks.map(
+      tvMediaBlock => {
+        return {
+          type: 'tvMedia',
+          model: {
+            ...tvMediaBlock,
+          },
+        };
+      },
+    );
+    it('Should process a On Demand TV block into a valid playlist item.', () => {
+      const result = buildSettings({
+        ...hindiTvBaseSettings,
+        blocks: hindiTvMediaBlocks as MediaBlock[],
+        pageType: MEDIA_PAGE,
+      });
+
+      expect(result).toStrictEqual({
+        playerConfig: {
+          product: 'news',
+          enableToucan: true,
+          appType: 'responsive',
+          autoplay: false,
+          externalEmbedUrl: '',
```

Comment:
According to the SMP Cookbook, this value is required if embedding is true. For the current test data, embedding is set to false, but there could be other examples where embedding is true. 

Let's ask our product team whether On Demand TV can be embedded (if it is even possible to configure this in the tooling for On Demand, TV  and if so, then we will need to include the embed URL. cc @gavinspence